### DISCUSSION TITLE
Edit Button on Github.com does not show for all files in branch

### DISCUSSION THREAD

**Discussion by BigTimeStats** [community/community#22952] [New to GitHub]
<p>Hello,</p>
<p>I am looking to edit a file in my Github repository: </p>
<p><a href="https://github.com/BigTimeStats/beach-volleyball" rel="nofollow noopener">https://github.com/BigTimeStats/beach-volleyball</a></p>
<p>Specifically, the file: <a href="https://github.com/BigTimeStats/beach-volleyball/blob/master/match_archive_2000_to_2017.csv" rel="nofollow noopener">https://github.com/BigTimeStats/beach-volleyball/blob/master/match_archive_2000_to_2017.csv</a></p>
<p>There is no edit button to edit it, meaning I have to delete/re-upload:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/97d53017c9ef491a69b61cd9af824e362a26f04a.png" data-download-href="/uploads/short-url/lFaQKXUalug0dztTlWHmPequgGK.png?dl=1" title="Screen Shot 2018-03-17 at 2.43.02 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/29850965/181074021-1b4affb6-1466-4ea7-95ad-cab0ccdb1c86.png" alt="Screen Shot 2018-03-17 at 2.43.02 AM.png" data-base62-sha1="lFaQKXUalug0dztTlWHmPequgGK" width="690" height="206" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/97d53017c9ef491a69b61cd9af824e362a26f04a_2_690x206.png, https://aws1.discourse-cdn.com/github/optimized/1X/97d53017c9ef491a69b61cd9af824e362a26f04a_2_1035x309.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/97d53017c9ef491a69b61cd9af824e362a26f04a_2_1380x412.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/97d53017c9ef491a69b61cd9af824e362a26f04a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-03-17 at 2.43.02 AM.png</span><span class="informations">1964×588 60.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>However, there is an edit button for my other file:</p>
<p><a href="https://github.com/BigTimeStats/beach-volleyball/blob/master/match_update_20170729_to_20170912.csv" rel="nofollow noopener">https://github.com/BigTimeStats/beach-volleyball/blob/master/match_update_20170729_to_20170912.csv</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/0509b337de45016c3e540cf2543cc4083eb5dbea.png" data-download-href="/uploads/short-url/Izawauzk5x3rwi9Ce7MRPacoCS.png?dl=1" title="Screen Shot 2018-03-17 at 2.43.35 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/29850965/181074022-c1e2cd84-af92-4936-9ff2-fe5a88f3aaa4.png" alt="Screen Shot 2018-03-17 at 2.43.35 AM.png" data-base62-sha1="Izawauzk5x3rwi9Ce7MRPacoCS" width="690" height="230" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/0509b337de45016c3e540cf2543cc4083eb5dbea_2_690x230.png, https://aws1.discourse-cdn.com/github/optimized/1X/0509b337de45016c3e540cf2543cc4083eb5dbea_2_1035x345.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/0509b337de45016c3e540cf2543cc4083eb5dbea_2_1380x460.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/0509b337de45016c3e540cf2543cc4083eb5dbea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-03-17 at 2.43.35 AM.png</span><span class="informations">1968×656 103 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The only difference I can see is the file size… however I have not been able to find any documentation on this being the case.</p>
<p>Thanks for the help!</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/bigtimestats">@bigtimestats</a>,</p>
<p>I’m not sure what the exact size is, but there is a limit to what our user interface can render and therefore what you can edit in the browser. However, you don’t necessarily need to delete and upload a new file every time that you want to edit this file. You could use a Git client like <a href="https://desktop.github.com/" rel="nofollow noopener">GitHub Desktop</a> to upload a new version of the file when it changes.</p>
<p>Hope this helps!</p>

**Comment by govindprabhu** [comment]
<p>But Github Desktop supports only 64 bit, I have a 32 bit machine. Any way out please suggest</p>

**Comment by TERRENCE2019** [comment]
<p>hAVING EXACT SAME ISSUE! I will post solution</p>
